### DISCUSSION TITLE
Example implementation w/ subscriptions and directives using Apollo Server

### DISCUSSION THREAD

**Discussion by theogravity** [hayes/pothos#264] [Show and tell]
Example using `giraphql` with directives, mutations, queries, and subscriptions, along with example unit and integration tests.

https://github.com/theogravity/graphql-giraphql-server-example

**Comment by hayes** [comment]
Thanks for sharing!  I haven't had time to look at it closely, but I've been thinking about trying to create a community resources page or something that has cool examples like this

**Comment by theogravity** [comment]
I updated the example to use the Pothos package name changes.

https://github.com/theogravity/graphql-pothos-server-example
